pkg/workloadmeta/collectors/internal/remote/process_collector/process_collector_test.go

```diff
@@ -233,11 +233,11 @@ func TestCollection(t *testing.T) {
 		},
 	}
 
+	mockConfig := config.Mock(t)
```

Comment:
I think that now that `streamHandler` accepts a config, it might be easier to declare a new config and use it:
```
conf := config.NewConfig("datadog", "DD", strings.NewReplacer(".", "_"))
conf.Set("language_detection.enabled", true)
```
I think with the mock is not so easy to ensure that no other goroutine is going to access it.

Reply:
Oh thank you ! It is precisely what I wanted to do but I did not know it existed